<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CmpWeight</class>
 <widget class="QWidget" name="CmpWeight">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1402</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>681</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Hash 技术比较次数：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>611</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>二分查找比较次数：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_hashCnt">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>90</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 28pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_biCnt">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>210</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 28pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>390</y>
     <width>681</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Hash 技术运行时间：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>520</y>
     <width>611</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>二分查找运行时间：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_hashTime">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>400</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 28pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_biTime">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>540</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 28pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>70</y>
     <width>81</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>次</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>190</y>
     <width>81</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>次</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>390</y>
     <width>81</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>秒</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>520</y>
     <width>81</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>秒</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
